docs/support.md

```diff
@@ -0,0 +1,120 @@
+# Currents Support & Maintenance Policy
+
+## Introduction
+
+This document outlines the support and maintenance policy for Currents. It is intended for customers deploying Currents in self-managed (on-prem) environments. The goal is to clearly define our responsibilities, provide guidance on components we support directly, and set expectations for areas where support is advisory or excluded.
+
+This document may be shared with customers and incorporated into future support material and onboarding guides.
+
+---
+
+## System Components & Support Scope
+
+Currents components fall into three categories:
+
+1. **Core (Fully Supported)**
+2. **Advisory Support**
+3. **Excluded (No Support)**
+
+---
+
+### 1. Core Components (Fully Supported)
+
+Once Currents is deployed and the correct setup is confirmed, Currents is responsible for the reliable and performant operation of the core services. This includes:
+
+- **Currents services running within the Kubernetes (K8S) cluster**
+- **Helm chart** configuration, parameters, and definitions
+- **Database configuration**:
+  - Indexes and query performance
+  - Data integrity and settings (e.g., backups, rollover policies)
+- **Redis configuration**:
```

Comment:
@twk3 not sure whether we'd include resources allocation for redis cluster

Reply:
we can leave it as is for now